### PR TITLE
Translate eloquent-serialization.md

### PR DESCRIPTION
**Pull request by neighborhood999**
#287
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/laravel-taiwan/docs/pull/309%23discussion_r48395414%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/309%23discussion_r48396675%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/309%23discussion_r48399752%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/309%23discussion_r48405855%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20d745352cbe96a2aded2cc9f42ea82e7cbe1d4e1a%20eloquent-serialization.md%20110%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/309%23discussion_r48395414%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu5982%5Cu679c%5Cu4f60%5Cu60f3%5Cu8b93%5Cu7d66%5Cu5b9a%5Cu6a21%5Cu578b%5Cu5be6%5Cu4f8b%5Cu5728%5Cu5e73%5Cu6642%5Cu88ab%5Cu96b1%5Cu85cf%5Cu7684%5Cu5c6c%5Cu6027%5Cu8b8a%5Cu70ba%5Cu53ef%5Cu898b%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A29%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cu5982%5Cu679c%5Cu4f60%5Cu60f3%5Cu8981%5Cu5728%5Cu7d66%5Cu5b9a%5Cu7684%5Cu6a21%5Cu578b%5Cu5be6%5Cu4f8b%5Cu4e2d%5Cu8b93%5Cu539f%5Cu5148%5Cu96b1%5Cu85cf%5Cu7684%5Cu5c6c%5Cu6027%5Cu8b8a%5Cu70ba%5Cu53ef%5Cu898b%5Cr%5Cn%5Cr%5Cn%40jigsawye%20%5Cu4f60%5Cu89ba%5Cu5f97%5Cu9019%5Cu6a23%5Cu6703%5Cu6bd4%5Cu8f03%5Cu901a%5Cu9806%5Cu55ce%5Cuff1f%22%2C%20%22created_at%22%3A%20%222015-12-24T05%3A28%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%2C%20%7B%22body%22%3A%20%22good%20%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-12-24T07%3A19%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%5Cuff01%22%2C%20%22created_at%22%3A%20%222015-12-24T09%3A40%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20eloquent-serialization.md%3AL81-104%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull d745352cbe96a2aded2cc9f42ea82e7cbe1d4e1a eloquent-serialization.md 110'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/309#discussion_r48395414'>File: eloquent-serialization.md:L81-104</a></b>
- <a href='https://github.com/jigsawye'></a> 如果你想讓給定模型實例在平時被隱藏的屬性變為可見
- <a href='https://github.com/neighborhood999'></a> 如果你想要在給定的模型實例中讓原先隱藏的屬性變為可見
@jigsawye 你覺得這樣會比較通順嗎？
- <a href='https://github.com/jigsawye'></a> good :+1:
- <a href='https://github.com/neighborhood999'></a> done！

<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/309?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/309?approve=1'></a>&nbsp;<a href='https://github.com/laravel-taiwan/docs/pull/309'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: eloquent-serialization.md [hunk 1]
```diff
@@ -81,23 +81,23 @@ Alternatively, you may use the `visible` property to define a white-list of attr
...
          * @var array
          */
         protected $visible = ['first_name', 'last_name'];
     }
 
-#### Temporarily Exposing Hidden Properties
+#### 暫時顯示隱藏的屬性
 
-If you would like to make some typically hidden attributes visible on a given model instance, you may use the `makeVisible` method. The `makeVisible` method returns the model instance for convenient method chaining:
+如果你想要讓一些通常被隱藏的屬性在你給定的模型的實例可以被看見，你可以使用 `makeVisible` 方法。`makeVisible` 方法回傳模型實例讓你可以方便的使用方法鏈結：
```
Comment (jigsawye) [inline eloquent-serialization.md]: 如果你想讓給定模型實例在平時被隱藏的屬性變為可見

Comment (neighborhood999) [inline eloquent-serialization.md]: 如果你想要在給定的模型實例中讓原先隱藏的屬性變為可見

@jigsawye 你覺得這樣會比較通順嗎？

Comment (jigsawye) [inline eloquent-serialization.md]: good :+1:

Comment (neighborhood999) [inline eloquent-serialization.md]: done！

**Pull request merged**